examples/EvaluatingChatAssistant/EvaluatingChatAssistant.ipynb

```diff
@@ -0,0 +1,353 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluating a Chat Bot\n",
+    "\n",
+    "This tutorial will walk through using Braintrust to evaluate a conversational, multi-turn chat assistant.\n",
+    "\n",
+    "These types of chat bots have become very important parts of applications, acting as customer service agents, sales representatives, or travel agents, to name a few. As an owner of such an application, it's important to be sure the bot provides value to the user.\n",
+    "\n",
+    "We will expand on this below, but the history and context of a conversation is usually crucial in being able to produce a good response. If you received a request to \"Make a dinner reservation at 7pm\" and you knew the context of where, on what date, and for how many people, you could provide some assistance; otherwise, you'd need to ask for more information. \n",
+    "\n",
+    "Before starting, please make sure that you have a Braintrust account. If you do not have one, you can [sign up here](https://www.braintrustdata.com)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installing dependencies\n",
+    "\n",
+    "Begin by installing the necessary dependencies if you have not done so already."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "powershell"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "npm install autoevals braintrust openai"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inspecting the data\n",
+    "\n",
+    "Let's take a look at the small dataset prepared for this cookbook. You can find the full dataset in the accompanying [dataset.ts](https://github.com/braintrustdata/braintrust-cookbook/tree/main/examples/EvaluatingCustomerSupportAgent/assets/dataset.ts) file. The `assistant` turns were generated using `gpt-4o`.\n",
+    "\n",
+    "Below is an example of a data point.\n",
+    "- `chat_history` contains the history of the conversation between the user and the assistant\n",
+    "- `input` is the last `user` turn that will be sent to the completion\n",
+    "- `expected` is the output expected given the input\n",
+    "\n",
+    "From looking at this one example, it is clear why we need the history to be able to provide a helpful response.\n",
+    "\n",
+    "If you were asked \"who won the men's trophy that year?\", you would wonder *What trophy? Which year?*. But if you read the `chat_history`, you'd be able to answer the question (maybe after some quick research)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "const dataset = [\n",
```

Comment:
I think ideally we would be able to run all the cells in the cookbook and it would produce the results that you've describe, so we probably want to run something like this:
```
import dataset from "./dataset.ts"
console.log(dataset.slice(0, 1));
```

(will need to set up tslab if you haven't yet)